cypress/e2e/shared/explorer.test.ts

```diff
@@ -1197,19 +1197,15 @@ describe('DataExplorer', () => {
 
         cy.getByTestID('save-as-dashboard-cell--submit').click()
 
-        // wait some time for save
-        cy.wait(200)
-        // ensure dashboard created with cell
-        cy.get('@org').then(({id: orgID}: Organization) => {
-          cy.fixture('routes').then(({orgs}) => {
-            cy.visit(`${orgs}/${orgID}/dashboards/`)
-            cy.getByTestID('dashboard-card--name')
-              .contains(dashboardCreateName)
-              .should('exist')
-              .click()
-            cy.getByTestID(`cell ${cellName}`).should('exist')
-          })
-        })
+        cy.location('pathname').should(
+          'match',
+          /^(?=.*dashboards)(?:(?!cell).)+$/
```

Comment:
is there any way this can be written out in a more approachable manner? this is dense even for a regex. it took me like 2 minutes to comprehend it, and even then, i'm not super confident that i understand it well. having to use regex101.com to check what this code tests isn't great. are you confident that this regex covers all your test cases?

but even after sorta understanding it, i'm not sure what is significant about this pattern. why do urls with dashboard but not cell in them matter?

Reply:
This was @drdelambre 's request, i am happy to change this but this is just a check that we get to the dashboards page and do not open an individual cell.

Reply:
can you explain that in a comment so someone reading this doesn't have to compute a regex in their head?